scielomanager/scielomanager/static/css/chosen.css

```diff
@@ -231,6 +235,7 @@ This file is generated by `grunt build`, do not edit it by hand.
 .chosen-container-multi .chosen-choices li.search-field input[type="text"] {
   margin: 1px 0;
   padding: 5px;
+  color: #CCC !important;
```

Comment:
> "This file is generated by `grunt build`, do not edit it by hand."

1) Não é uma boa pratica modificar os arquivos dos plugins, porque quando o plugin for atualizado, estas mudanças possivelmente vão se perder. Ou pode complicar o trabalho de atualização (no caso substituir este arquivo css na integra por um mais atualizado)
2) Modificações neste nível, vão afetar todas a instâncias aonde este plugin é utilizado. Tem certeza que deve ser assim? ou são modificações ad-hoc para esta nova interface de relatórios?
3) enquanto ao uso do ``!important`` tem pessoas que não gostam nem um pouco disso! [1], mas a verdade é que na maioria das vezes pode ser evitado, com diz aqui [2] e [3].

[1] https://twitter.com/stefsull/status/70631020352913408 
[2] https://css-tricks.com/when-using-important-is-the-right-choice/
[3] https://stackoverflow.com/questions/3427766/should-i-avoid-using-important-in-css